com.sublimehq.SublimeText.appdata.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Endless Mobile, Inc. -->
+<component type="desktop">
+  <id>com.sublimehq.SublimeText</id>
```

Comment:
this is not consistent with this: https://github.com/flathub/com.sublimetext.three

Reply:
Reasoning is discussed here: https://github.com/flathub/com.sublimetext.three/issues/26

Even though people call it Sublime Text 4, it is really just 'Sublime Text'. There are no plans for a major new version, just continual updates to this with a new pricing model.

Reply:
but all the links point to `sublimetext.com`.... include the download and the EULA

Reply:
I could include links from here:
https://www.sublimehq.com/

Reply:
It appears I had the issues link incorrect, it lives under the `sublimehq` GitHub:
https://github.com/sublimehq/